packages/client/hmi-client/src/components/data-explorer/search-bar.vue

```diff
@@ -16,97 +15,79 @@
 				<Menu ref="autocompleteMenu" :model="autocompleteMenuItems" :popup="true"> </Menu>
 				<i
 					class="pi pi-times clear-search"
-					:class="{ hidden: isClearSearchButtonHidden }"
+					:class="{ hidden: isClearQueryButtonHidden }"
 					style="font-size: 1rem"
-					@click="clearText"
-				></i>
+					@click="clearQuery"
+				/>
 			</span>
-			<i class="pi pi-history" />
-			<i class="pi pi-image" title="Search by Example" @click="toggleSearchByExample" />
+			<!-- <i class="pi pi-history" title="Search history" /> -->
+			<!-- <i class="pi pi-image" title="Search by Example" @click="toggleSearchByExample" /> -->
 		</div>
-		<span class="suggested-terms" v-if="suggestedTerms && suggestedTerms[0]"
-			>Suggested terms:<Chip
-				v-for="item in suggestedTerms"
-				:key="item"
-				removable
-				remove-icon="pi pi-times"
-			>
-				<span @click="addSearchTerm(item)">{{ item }}</span>
-			</Chip>
-		</span>
-	</div>
+	</section>
 </template>
 
 <script setup lang="ts">
-import { computed, onMounted, ref, watch } from 'vue';
+import { computed, onMounted, ref } from 'vue';
 import { useRoute } from 'vue-router';
 import InputText from 'primevue/inputtext';
-import Chip from 'primevue/chip';
 import * as EventService from '@/services/event';
-import { EventType } from '@/types/EventType';
 import useResourcesStore from '@/stores/resources';
 import { ResourceType } from '@/types/common';
 import { getRelatedWords, getAutocomplete } from '@/services/data';
 import Menu from 'primevue/menu';
+import { EventType } from '@/types/EventType';
+
+const emit = defineEmits(['query-changed', 'toggle-search-by-example']);
+
+const route = useRoute();
+const resources = useResourcesStore();
 
 const props = defineProps<{
-	text?: string;
-	suggestedTerms?: string[];
+	text: string;
 	resultType: string;
```

Comment:
I want all the `resultType`s to be named `resourceType` eventually. Also you can make it of type `ResourceType`

Reply:
@edwin what is the `resultType` doing?

Reply:
`resulttype` is passed from `dataexplorer`, it's whether or not the user is on papers, models, or datasets; for the latter two, there is no call to the suggestions api since it requires a dataset parameter, which is not present for the latter two

Reply:
... this will need to be reviewed with @pascaleproulx and @hvasque14, because that's an UX issue.

Reply:
@shawnyama I will not change this, because resources and results are still.